### PR TITLE
Show a page suggesting creating a policy after resource is created

### PR DESCRIPTION
**Pull request by AndrewDryga**

### CODE REVIEW

**Comment by jamilbk** [comment]
Feeback from standup:

- Move help page to Show Resource
- Add Policies column to Resources table

**Review by bmanifold — APPROVED** [review]

**Review by jamilbk — CHANGES_REQUESTED** [review]
Just copying feedback here from standup yesterday:

- After Create/Update, redirect to Show Resource page with a notice if there are no policies yet
- Add Policies column to Resources table to show policies count

**Comment by AndrewDryga** [comment]
Screenshot 2023-11-10 at 11 19 37
Screenshot 2023-11-10 at 11 31 45
Screenshot 2023-11-10 at 11 33 30

**Comment by AndrewDryga** [comment]
@bmanifold I also hidden the traffic filters from this page using the feature flag, I think it was a leftover from your PR.

File: elixir/apps/web/lib/web/live/policies/new.ex [hunk 1]
```diff
@@ -83,20 +86,37 @@ defmodule Web.Policies.New do
...
   def handle_event("submit", %{"policy" => policy_params}, socket) do
+    policy_params = put_default_policy_params(policy_params, socket)
+
     with {:ok, policy} <- Policies.create_policy(policy_params, socket.assigns.subject) do
-      {:noreply, redirect(socket, to: ~p"/#{socket.assigns.account}/policies/#{policy}")}
+      if site_id = socket.assigns.params["site_id"] do
+        {:noreply,
+         push_navigate(socket, to: ~p"/#{socket.assigns.account}/sites/#{site_id}?#resources")}
+      else
+        {:noreply, redirect(socket, to: ~p"/#{socket.assigns.account}/policies/#{policy}")}
```
Comment (bmanifold) [inline elixir/apps/web/lib/web/live/policies/new.ex:106]: Just curious for my own edification, was `redirect` chosen for a reason here?  Won't this do a whole page reload?

Comment (AndrewDryga) [reply elixir/apps/web/lib/web/live/policies/new.ex:106]: Good catch, it's a copy-paste. We have it in a ton of places, let's get this PR merged and I'll fix all the occurences in the upcoming PR.

Comment (bmanifold) [reply elixir/apps/web/lib/web/live/policies/new.ex:106]: Sounds good 👍

**Pull request merged**